users/miketronic/process_records.c

```diff
@@ -0,0 +1,134 @@
+// Copyright 2023 @miketronic -- Mike B <mxb540@gmail.com>
+// SPDX-License-Identifier: GPL-2.0+
+
+#include "miketronic.h"
+#include "process_records.h"
+
+__attribute__((weak)) bool process_record_keymap(uint16_t keycode, keyrecord_t *record) {
+    return true;
+}
+
+__attribute__((weak)) bool process_record_secrets(uint16_t keycode, keyrecord_t *record) {
+    return true;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+#ifdef CONSOLE_ENABLE
+    uprintf("KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
+#endif
+
+    switch (keycode) {
```

Comment:
just a heads up, if you want the keymap/etc version to overwrite stuff, you could use: 

```c
    if (!(process_record_keymap(keycode, record) && process_record_secrets(keycode, record)) {
        return false;
    }
```

Reply:
I honestly don't know what this does, maybe it was part of the old "MAKE" macro that is no longer needed.  Omitted this.